src/modules/CreateChannel/context/CreateChannelProvider.tsx

```diff
@@ -39,7 +39,12 @@ export interface CreateChannelContextInterface {
   createChannel: CreateChannel;
   sdk: SendbirdGroupChat;
   userListQuery?(): UserListQuery;
+  /**
+   * @deprecated
+   * Use the onClickCreateChannel instead
+   */
   overrideInviteUser?(params: OverrideInviteUserType): void;
+  // TODO: onClickCreateChannel instead of overrideInviteUser
```

Comment:
TODO: Rename the overrideInviteUser to onClickCreateChannel (in the CreateChannel)